frontend/containers/discover-map/component.tsx

```diff
@@ -138,22 +150,28 @@ export const DiscoverMap: FC<DiscoverMapProps> = ({ onSelectProjectPin }) => {
               onChange: handleChangeVisibleLayer,
             }}
           />
-          <LocationSearcher
-            className="pointer-events-auto"
-            onLocationSelected={handleLocationSelected}
-          />
-        </div>
-
-        <div className="absolute top-4 right-4">
-          <MapHelp />
         </div>
 
-        <Controls className="absolute h-fit max-h-[45%] bottom-4 right-4 overflow-y-auto">
-          <LayerLegend
-            className="bg-white"
-            onCloseLegend={(layerGroup) => resetField(layerGroup as keyof MapLayersSelectorForm)}
-            layersLegends={layerLegends}
-          />
+        <Controls className="w-full h-full">
```

Comment:
The buttons have different hover and focus styles:
- when hovered, the two zoom buttons change the icon colour to dark green and don't have any border
- when focused, the map layers and help button use a ring

Can we have the same behaviour on all the buttons? It doesn't matter which one you pick.

https://user-images.githubusercontent.com/6073968/193027335-13b1d340-9fd9-4a08-8a0c-1847db6c80c6.mp4

Reply:
I don't think you've addressed all the styles inconsistency yet.

**Hover**

The “Map layers” button has a different hover style:

![The button has a green ring when hovered](https://user-images.githubusercontent.com/6073968/193222807-e666d1bc-c372-4d0c-8041-62e377c04aa7.png)

It should look like this to match the other buttons:

![The button has a green text and no ring](https://user-images.githubusercontent.com/6073968/193223235-fdd3ab6e-9ed4-4119-b0dd-145531f503d3.png)

You can simply remove the current `hover:*` classes and add `hover:text-green-dark transition-all` instead.

**Focus**

The “Map layers” button has also a different focus style:

![The button has a thick green ring when focused](https://user-images.githubusercontent.com/6073968/193223728-9cf4ae7c-ef34-499f-958b-da60883e690e.png)

To match the other buttons, it should look like this:

![The button's ring is less think and offset by a few pixels](https://user-images.githubusercontent.com/6073968/193223998-1232f4f9-1b1d-474f-9c75-dbc453dd93b1.png)

To achieve this, remove all the previous `focus-visible:*` classes and replace them by `focus-visible:outline-green-dark focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2`.

The styles of the search component also don't match the rest:

![The search component has a thick green ring when focused](https://user-images.githubusercontent.com/6073968/193225656-1cc98e40-f2d1-43aa-8966-c905f598c1ac.png)

I think it should look like this:

![The search component has a slimmer green ring offset by a few pixels](https://user-images.githubusercontent.com/6073968/193225452-2b9ecf75-cbf3-4875-ab61-09c7acd051e3.png)

You can achieve this by applying these classes to the `<div />` that has the `onMouseEnter` callback when it's open and focused: `outline outline-green-dark outline-2 outline-offset-2`.